src/vsr/superblock.zig

```diff
@@ -108,19 +114,35 @@ pub const SuperBlockSector = extern struct {
         /// The view number of the replica.
         view: u32,
 
+        reserved: [8]u8 = [_]u8{0} ** 8,
+
         comptime {
-            assert(@sizeOf(VSRState) == 24);
+            assert(@sizeOf(VSRState) == 48);
             // Assert that there is no implicit padding in the struct.
             assert(@bitSizeOf(VSRState) == @sizeOf(VSRState) * 8);
         }
 
+        pub fn root(cluster: u32) VSRState {
+            return .{
+                .commit_min_checksum = vsr.Header.root_prepare(cluster).checksum,
+                .commit_min = 0,
+                .commit_max = 0,
+                .view_normal = 0,
+                .view = 0,
+            };
+        }
+
         pub fn internally_consistent(state: VSRState) bool {
```

Comment:
`internally_consistent` and `monotonic` are only ever called inside asserts. Would it make more sense for them to be `verify_internally_consistent` and `verify_monotonic` and assert internally? As a bonus, we'd then get to see exactly which assert failed.

Reply:
We do use `monotonic()` as a boolean in `superblock_quorums.zig`, when we are trying to pick the working superblock quorum.

Also, one benefit of returning `bool` is it is more amenable to testing (at least until Zig supports testing panics). (We don't have these here, but we probably should since they are so critical for correctness.)